camunda-forim-io-vue/src/components/TasklistSortoptions.vue

```diff
@@ -1,27 +0,0 @@
-<template>
```

Comment:
suggestion => rename  "TasklistSortoptions.vue"   to    "TasklistSortOptions.vue"